templates/accounts/delete_confirmation.html

```diff
@@ -16,7 +16,7 @@
 
 {% block content %}
 {% for anon in anonymised %}
-  <p>The User "{{anon.name}}" will be anonymised</p>
+<p>The User "{{anon.name}}" will be {% if anon.deleted %}deleted{% else %}anonymised{% endif %}</p>
```

Comment:
This I don't understand, when is this shown and in which cases do we see deleted or anonymised?